intelmq/bots/experts/asn_lookup/expert.py

```diff
@@ -57,5 +69,89 @@ def check(parameters):
         except Exception as exc:
             return [["error", "Error reading database: %r." % exc]]
 
+    @classmethod
+    def run(cls, parsed_args=None):
+        if not parsed_args:
+            parsed_args = cls.argparser.parse_args()
+
+        if parsed_args.update_database:
+            cls.update_database()
+
+        else:
+            super().run(parsed_args=parsed_args)
+
+    @classmethod
+    def update_database(cls):
+        bots = {}
+        runtime_conf = load_configuration(RUNTIME_CONF_FILE)
+        try:
+            for bot in runtime_conf:
+                if runtime_conf[bot]["module"] == __name__:
+                    bots[bot] = runtime_conf[bot]["parameters"]["database"]
+
+        except KeyError as e:
+            print("Database update failed. Your configuration of {0} is missing key {1}.".format(bot, e))
+            sys.exit(1)
```

Comment:
combine with previous line, the the message is printed to stderr

Reply:
Done.